test/runtests.jl

```diff
@@ -258,18 +259,18 @@ end
 end
 
 @testset "full-text search" begin
-    function linear_search(query)
+    global function linear_search(query)
```

Comment:
Why is this needed?

Reply:
@gmassman

Reply:
That was fixing a deprecation warning from julia 0.7... I'll remove it and pass in `textstr` so no need for global.